scripts/derive_galileo_gas_parameter/derive_galileo_gas_parameter.py

```
@@ -0,0 +1,1465 @@
+    cur_slot = cur_slot - (cur_block - block_number) + 1
+    cur_block = block_number
+
+    found_the_blob = False
+    while not found_the_blob:
+        cur_slot -= 1
+        url = f"{mainnet_beacon_url}/eth/v1/beacon/blob_sidecars/{cur_slot}"
+
+        headers = {'accept': 'application/json'}
+        try:
+            response = requests.get(url, headers=headers, timeout=5)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)